skipruntime-ts/native/src/Runtime.sk

```diff
@@ -984,11 +984,8 @@ class Collection(hdl: SKStore.EHandle<JSONID, JSONFile>) {
       ),
       None(),
       Array[
-        SKStore.DirSub(
-          this.hdl.getDirName(),
-          "",
-          SKStore.OJSON(Array[]),
-          (__) ~> None(),
+        SKStore.DirSub(this.hdl.dirName, "", SKStore.OJSON(Array[]), (__) ~>
```

Comment:
Out of curiosity, any reason why `(__) ~>` uses a double underscore as param name?

Reply:
Seems to have been introduced in 618f0adf1fec83f9646a60c69b5f8a81ddb5b2ce, ask @skiplabsdaniel